lib/vagrant-libvirt/action/package_domain.rb

```diff
@@ -38,7 +38,7 @@ def call(env)
           `qemu-img rebase -p -b "" #{@tmp_img}`
           # remove hw association with interface
           # working for centos with lvs default disks
-          `virt-sysprep --no-logfile --operations defaults,-ssh-userdir -a #{@tmp_img}`
+          `virt-sysprep --no-logfile --operations defaults,-ssh-userdir,-ssh-hostkeys -a #{@tmp_img}`
```

Comment:
It would be more useful to allow a use to provide some options, but also the problem with including this is that after packaging, `vagrant up` should result in different hostkeys being generated for each run.

If you can change this to read from the environment maybe `VAGRANT_LIBVIRT_VIRT_SYSPREP_OPERATONS` is a reasonable option to use?

Something like the following seems like it should work?
```
          operations = ENV.get('VAGRANT_LIBVIRT_VIRT_SYSPREP_OPERATONS', 'defaults,-ssh-userdir')
          `virt-sysprep --no-logfile --operations #{operations} -a #{@tmp_img}`
```

Reply:
Will also want to update the README.md to mention this being available, it doesn't really solve the problem because the ssh hostkeys should get re-created, but at least it means that anyone hitting various problems around the packaging with virt-sysprep can adjust as needed.